src/Core/OrganizationFeatures/OrganizationSubscriptions/Interface/IAddSecretsManagerSubscriptionCommand.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Bit.Core.Entities;
+
+namespace Bit.Core.OrganizationFeatures.OrganizationSubscriptions.Interface;
+
+public interface IAddSecretsManagerSubscriptionCommand
```

Comment:
Please add an XML doc comment to clarify that this is only for adding SM to an **existing** organization.

Reply:
no longer required, see below

Reply:
resolved